docs/src/content/core/voice.mdx

```diff
@@ -318,3 +324,5 @@ to help them through it in a friendly way.
     Eimerman, +2
   </Quote>
 </EqualColumnLayout>
+
+<br />
```

Comment:
no idea why this works....do i need to clearfix flex containers now...

Reply:
inline scripts, body styles, now this...